helms/legion/templates/edge/edge-server.yaml

```diff
@@ -42,9 +42,13 @@ spec:
           value: "127.0.0.1"
         - name: FLUENTD_PORT
           value: "24224"
+        - name: FEEDBACK_COLLECTOR_BACKEND
```

Comment:
can we use a common name for feedback collector\aggregator since this is a single entity?

Reply:
done